cloud/spawn.go

```diff
@@ -23,6 +23,14 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Valid key types.
+const (
+	rsa     = "ssh-rsa"
```

Comment:
nit: can these be called `pubKeyRSA`, `pubKeyDSS`, etc. since it's basically an enum of strings?

Reply:
done